docker/cypress/Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM cypress/included:5.2.0
 
 WORKDIR /root
 
+RUN npm install "cypress-failed-log"
```

Comment:
This will always pull in the latest, which might not be the best idea considering all the recent JS CVEs. You could either set the version in the install command or create a package.json and install from there.

Reply:
True. I'll fix the version here for now